app/views/users/index.html.erb

```diff
@@ -1,6 +1,9 @@
 <% title _("Users") %>
 
-<% title_actions new_link(_("Create User")) %>
+<% title_actions new_link(_("Create User")), new_link(_("Invalidate JWTs for all users"),
```

Comment:
The `Invalidate JWTs for all users` button should not be primary IMO.
![Screenshot from 2025-01-21 07-58-02](https://github.com/user-attachments/assets/1be51a02-3c57-4a70-85c1-1556d5ed5ed5)